### ISSUE TITLE
Error when parsing @throws

### ISSUE THREAD

**Issue by fjgarlin** [code-lts/doctum#35]
When parsing the `@throws` tag without summary like [here](https://git.drupalcode.org/project/drupal/blob/9.3.x/core/lib/Drupal/Core/ImageToolkit/ImageToolkitOperationBase.php#L101-102)
```
   * @throws \InvalidArgumentException.
   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException.
```
the application throws the following exception:
```
In class.twig line 316:
  An exception has been thrown during the rendering of a template ("Warning: Cannot assign an empty string to a string offset in phar:///.../doctum.phar/src/Reflection/MethodReflection.php line 106").                                                                                                          
                                                                                                                                                                                           
In ErrorHandler.php line 42:
  Warning: Cannot assign an empty string to a string offset in phar:///.../doctum.phar/src/Reflection/MethodReflection.php line 106
```
and does not finish the documentation generation.

**Comment by williamdes** [comment]
Thank you for the report and for the fix @fjgarlin !

**Comment by williamdes** [comment]
I added tests in b18c7b7ebc4bd896af195f7a1e86e5b9cb58f5a6 and 5.3 series are non affected, the fix will target 5.4 series
